jobs/build/buildvm-maint/Jenkinsfile

```diff
@@ -38,10 +38,10 @@ node('openshift-build-1') {
 
                 stage("push images") {
                     dir ( "enterprise-images" ) {
-                        sh './oit/oit.py --user=ocp-build --group sync-misc distgits:push-image --to-defaults'
-                        sh './oit/oit.py --user=ocp-build --group sync-3.7 distgits:push-image --to-defaults'
+                        sh './oit/oit.py --user=ocp-build --group sync-misc images:push --to-defaults'
```

Comment:
images:push is for distgit in current PR. Maybe change in oit PR and make images:push-distgit do the distgit push?